docs/Releasing.md

```diff
@@ -27,4 +27,167 @@ After checking that everything looks good, you just need to push the changes:
+<p>o Create a new branch in the main WP apps (WordPress-iOS, WordPress-Android) named <code>gutenberg/integrate_release_X.XX.X</code>.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Use the commit hash of the head of the release branch in Gutenberg-Mobile as the reference for the main apps.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Create a PR in WPAndroid and WPiOS. Example on <a href="https://github.com/wordpress-mobile/WordPress-Android/pull/10863">WPAndroid</a>, <a href="https://github.com/wordpress-mobile/WordPress-iOS/pull/13010">WPiOS</a>. To write the PR description: pull the differences between previous and current releases. Update the RELEASE-NOTES.txt files in each PR (get info from the gutenberg-mobile RELEASE-NOTES.txt).</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o In iOS update the file <code>Podfile</code> to point to the new hash in GB-Mobile and if needed also update the reference to Aztec to the new release. Then run <code>rake dependencies</code>, commit and push.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o On Android update the git submodule reference for <code>libs/gutenberg-mobile</code> (<code>cd libs/gutenberg-mobile &amp;&amp; git checkout release/X.XX.X &amp;&amp; git pull origin release/X.XX.X &amp;&amp; cd .. &amp;&amp; git add gutenberg-mobile</code>) and run <code>python tools/merge_strings_xml.py</code> to update the main strings.xml file.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Create new branches <code>gutenberg/after_X.XX.X</code> in WPAndroid and WPiOS and keep them up to date with the release branches. These are to be doubles for develop on the main apps for mobile gutenberg dev’s WP app PR’s that didn’t or shouldn’t make it into the X.XX.X editor release.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"level":3} -->
+<h3>New Aztec Release</h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>o Make sure there is no pending Aztec PR required for this Gutenberg release. Check the commit hash referred in the gutenberg repo is in the Aztec <code>develop</code> branch. If it's not, make sure pending PRs are merged before next steps.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Open a PR on Aztec repo to update the <code>CHANGELOG.md</code> and <code>README.md</code> files with the new version name.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>o Create a new release and name it with the tag name from step 1. For Aztec-iOS, follow <a href="https://wp.me/p77Llu-buw">this post</a>. For Aztec-Android, releases are created via the <a href="https://github.com/wordpress-mobile/AztecEditor-Android/releases">GitHub releases page</a> by hitting the “Draft new release” button, put the tag name to be created in the tag version field and release title field, and also add the changelog to the release description. The binary assets (.zip, tar.gz files) are attached automatically after hitting “Publish release”.</p>
```

Comment:
Do you think we could move the information about creating an Aztec-iOS release to somewhere in the Aztec-iOS repo @SergioEstevao ? That would allow us to update this to be a public link.

Reply:
Something like this: https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1294

Reply:
Thanks @SergioEstevao ! I approved that PR with a couple of minor comments. I also went ahead and updated this to refer to that doc in the Aztec-iOS repo in 73796b4.

Reply:
Here is the final link: https://github.com/wordpress-mobile/AztecEditor-iOS/blob/develop/Documentation/ReleaseProcess.md